ui/tests/integration/components/replication-secondary-card-test.js

```diff
@@ -69,7 +69,7 @@ module('Integration | Enterprise | Component | replication-secondary-card', func
   test('it renders hasErrorMessage when state is idle', async function(assert) {
     this.set('stateError', STATE_ERROR);
     await render(
-      hbs`<ReplicationSecondaryCard @replicationDetails={{stateError}} @title={{title}} @hasErrorClass={{true}}>`
+      hbs`<ReplicationSecondaryCard @replicationDetails={{stateError}} @title={{title}} @hasErrorClass={{true}} />`
```

Comment:
i might've missed your response, but why does `hasErrorClass` have to be manually set?

Reply:
I redid somethings here and then missed this part.  Fixed.